src/components/modals/addMovie/addMovieModal.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { ErrorMessage, Field, Form, Formik } from 'formik'
+
+import { postMovieRequested } from '../../../redux/actions'
+
+import styles from './addMovieModal.module.css'
+
+export const AddMovieModal: React.FC = () => {
+  const dispatch = useDispatch()
+
+  const initialValues = {
+    title: '',
+    release_date: '',
+    poster_path: '',
+    genres: [''],
+    overview: '',
+    runtime: 0,
+  }
+
+  const handleSubmit = (movie: any) => {
```

Comment:
avoid using `any`

Reply:
Also, add that after sending a request modal closes, or show any message to a user that the request was sent etc.